### ISSUE TITLE
Email: ITI Comments on First Public Working Draft of W3C Accessibility Guidelines (WCAG) 3.0 (9 - Requirement for automation only level)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#451]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0012.html): 
from [PDF](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/att-0012/ITI_Comments_on_WCAG_3.0_First_Public_Working_Draft_20210224_1_.pdf)

1.	Are there additional Design Principles and Requirements that should be included in the WCAG 3 project?

Yes - inclusion of a compliance level or mechanism that only depends upon automated/automatable tests rather than needing human judgement and review.  Sites need to be able to approach some level of conformance without manual effort, or with minimal manual effort on a limited number of key outcomes. This would, of course, not be a desired end-point for any content, but this is sufficiently important for many large and highly dynamic sites that it merits mention in the requirements document.

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by jspellman** [comment]
Draft Response: We have received similar requests for an automated test level.  There are some complex issues associated with this, including the problems of reducing the accessibility at that level.  We intend to make WCAG3 testing more granular to support automated testing.  We are going to address this issue when we have a more detailed conversation on Conformance.

**Comment by alastc** [comment]
The response above was approved by the group.
